packages/core/src/modules/dids/methods/sov/IndySdkSovDidRegistrar.ts

```diff
@@ -24,15 +25,15 @@ export class IndySdkSovDidRegistrar implements DidRegistrar {
     const didRepository = agentContext.dependencyManager.resolve(DidRepository)
 
     const { alias, role, submitterDid, indyNamespace } = options.options
-    const seed = options.secret?.seed
+    const privateKey = options.secret?.privateKey
 
-    if (seed && (typeof seed !== 'string' || seed.length !== 32)) {
+    if (privateKey && (typeof privateKey !== 'object' || privateKey.length !== 32)) {
```

Comment:
In addition to validting in the wallet, we could also look at creating util methods `isValidSeed`, `isValidPrivateKey`. But I'm not sure if the seed  / privateKey is always the same for each key type...